conf/app.routes

```diff
@@ -12,7 +12,6 @@ HEAD       /traders/:eori/records/store             uk.gov.hmrc.tradergoodsprofi
 GET        /traders/:eori/records/count             uk.gov.hmrc.tradergoodsprofilesdatastore.controllers.GetRecordsController.getRecordsCount(eori: String)
```

Comment:
can we check whether we need line 11 and delete it and all the code related to it too? And can you drop a note in the ticket if you do this.

Reply:
This will be in a future ticket, Beth says :)

Reply:
https://jira.tools.tax.service.gov.uk/browse/TGP-2656